src/main/java/net/datafaker/Internet.java

```diff
@@ -179,7 +179,7 @@ public String macAddress() {
      * @return a correctly formatted IPv4 address.
      */
     public String ipV4Address() {
-        return String.format("%d.%d.%d.%d",
+        return String.format(faker.getLocale(), "%d.%d.%d.%d",
```

Comment:
I don't understand why you'd want to use the locale to format an ip address.

Reply:
finally it will be rewritten by https://github.com/datafaker-net/datafaker/pull/56